app/src/main/java/com/kickstarter/ui/activities/BackingActivity.java

```diff
@@ -22,7 +22,10 @@ public void onCreate(final @Nullable Bundle savedInstanceState) {
 
     this.viewModel.outputs.showBackingFragment()
             .compose(bindToLifecycle())
-            .subscribe(this::startBackingFragment);
+            .subscribe(it -> {
+              this.startBackingFragment(it);
+              this.backingFragment().pageViewed();
```

Comment:
No need to trigger the page Viewd event here, it should be fired on the BackingFragment